src/main/java/org/folio/qm/converter/field/dto/CommonDataFieldConverter.java

```diff
@@ -54,6 +55,7 @@ private String convertIndicator(char ind) {
   private Optional<UUID> extractAuthorityId(List<Subfield> subfields) {
```

Comment:
This converter is a common one so, we can't add here any logic related to only authorities. So, I suggest creating an additional converter for Authority type records.

Reply:
I think $9 field will come in all "MARC Authority", "MARC Bibliographic", "MARC Holdings" records, right? If yes, should I keep the logic as it is?